add some changes

artursmolin opened the pull request:
1.Add timezone + language
2.Add about.html(+ a href links)
3.Add /author/about.html( + a href links)
4.Rewrite Readme.RD
5.Add module with Version of app

artursmolin committed:
add some changes

1.Add timezone + language
2.Add about.html(+ a href links)
3.Add /author/about.html( + a href links)
4.Rewrite Readme.RD
5.Add module with Version of app

artursmolin committed:
Update version.rb

max-underthesun commented on app/config/application.rb:
```diff
@@ -10,6 +10,8 @@ module App
   class Application < Rails::Application
```
вообще это приложение должно называться `TestGuru` (у тебя и в Ридми так написано)
при создании нужно было следующее написать: `rails new test_guru` и тогда модуль приложения назывался бы `TestGuru`
создание проекта освещено в самом начале 4-го скринкаста ("Знакомство с Rails. Практика")

у тебя же (в 9 строке) видим `module App` - не очень говорящее название ))

max-underthesun commented on app/config/initializers/version.rb:
```diff
@@ -1 +1,5 @@
-VERSION = '0.1.0'
+module MySite
```
здесь конечно было бы верным поместить твой инишиалайзер именно в контекст твоего приложения или другими словами в модуль твоего приложения

т.е. здесь нужно указать `module TestGuru`
ну или, если оставишь свое приложение "как есть" (не очень конечно верное решение), то `module App`

max-underthesun commented on app/config/initializers/version.rb:
```diff
@@ -1 +1,5 @@
-VERSION = '0.1.0'
+module MySite
+  class Application
+    VERSION = "0.1.1"
```
не вижу изменений по этому комментарию: https://github.com/drujok/Rails/pull/2#discussion_r270118071
нужно использовать метод `.freeze`

max-underthesun commented on app/config/application.rb:
```diff
@@ -10,6 +10,8 @@ module App
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
+    config.time_zone = "Moscow"
+    config.i18n.default_locale = :'en-GB'
```
ТЗ: "Добавьте в конфигурацию Rails приложения своё местоположение: часовой пояс и язык."
давай все таки держаться ближе к ТЗ
если локация Москва, то язык - русский
(ну и потом будет лучше именно русский иметь дефолтным в заданиях по локализации)

max-underthesun replied:
второй момент
вот [документация](https://guides.rubyonrails.org/i18n.html#configure-the-i18n-module) по конфигурированию "дефолтной локали" в рельсах
обрати внимание: основной вариант работы с локалью включает только указания языка, регион не указывается

max-underthesun commented on app/public/about.html:
```diff
@@ -0,0 +1,19 @@
+    <!-- This file lives in public/500.html -->
+    <div class="dialog">
+      <div>
+        <a href="www.thinknetica.com">Школа Thinknetica</a>
```
отлично, ссылка появилась
есть одно НО: ссылки на сторонние ресурсы правильно открывать в новом окне (добавь соответствующую опцию тегу)

max-underthesun commented on app/public/about/author.html:
```diff
@@ -0,0 +1,18 @@
+    <!-- This file lives in public/500.html -->
+    <div class="dialog">
+      <div>
+        <a href="https://github.com/drujok">Github Profile: Drujok</a>
```
нужно открывать в новом окне

artursmolin committed:
fix some

1.Ref language
2.Add target = "_blank" to html about,author
3. Add freeze meth to version.rb
4. Change App to TestGuru
